src/settings/MappingProfiles/MappingProfilesTransformationsModal/TransformationsField/TransformationsField.js

```diff
@@ -73,25 +72,25 @@ export const TransformationField = ({
       isEqual={isEqual}
     >
       {({ fields }) => (
-        <MultiColumnList
-          id="mapping-profiles-form-transformations"
-          fields={fields}
-          contentData={prepareData(fields.value)}
-          totalCount={fields.value.length}
-          columnMapping={columnMapping}
-          columnWidths={columnWidths}
-          visibleColumns={visibleColumns}
-          formatter={formatter}
-          autosize={autosize}
-        />
+        <div className={css.mappingProfilesFormTransformations}>
+          <MultiColumnList
+            id="mapping-profiles-form-transformations"
+            fields={fields}
+            contentData={contentData || fields.value}
+            totalCount={fields.value.length}
```

Comment:
I wonder, if we use search result (For example, we display only 2 of 10 items), so should we apply the number of all items here in this case? I'm not sure but property can used to calculate the list height so this is why we have to apply some additional styles to fix it. @VictorSoroka1 what do you think about it?

Reply:
Regarding showing of a number of total items - I looked through the experience of how it is done in other places and found that only the amount of found items is shown. We can suggest this as a further improvement to the PO on grooming.

Regarding the height, I would not use the number of items for calculation of height as the height of each row can vary if there is too much content in the row.

Reply:
Turned out styles can be removed as the `autozise` prop for MCL can now be removed after the search pane visibility is manged via CSS. `autozise` was used solely to handle the case when the search pane hides and thus the MCL recalculates the column widths. The way `autozise` does it is by setting the width and height on the mcl container which should be overridden in our case for the wrong calculation of height when the list renders not all possible transformations rows. For some reason, it worked incorrectly when search pane was hidden via conditional rendering in the component. Thanks for the comment which allowed me to rethink this idea once again.